flytekit/types/file/file.py

```diff
@@ -324,6 +325,60 @@ def assert_type(
     def get_literal_type(self, t: typing.Union[typing.Type[FlyteFile], os.PathLike]) -> LiteralType:
         return LiteralType(blob=self._blob_type(format=FlyteFilePathTransformer.get_format(t)))
 
+    def get_mime_type_from_python_type(self, extension: str) -> str:
+        extension_to_mime_type = {
+            "html": mimetypes.types_map[".html"],
```

Comment:
could we append `mimetypes.types_map` to the `extension_to_mime_type`?

something like
```python
for extension, mimetype in range(mimetypes.types_map.items()):
    extension_to_mime_type[k.split(".")[1]] = mimetype
```

Reply:
btw, some tests are failing

Reply:
updated and all tests passed

Reply:
![Screenshot 2023-11-07 at 11 43 50 AM](https://github.com/flyteorg/flytekit/assets/14800485/d98c0c6f-f7c2-4179-9a02-59a37790ffb9)